Gruntfile.js

```diff
@@ -299,7 +307,7 @@ module.exports = (grunt) => {
     const fork2 = function () {
       grunt.log.writeln('backend: forking...')
       child = fork(backendIndex, process.argv, {
-        env: process.env,
+        env: { ...process.env, GI_VERSION },
```

Comment:
Why was this modified? This doesn't look correct, `GI_VERSION` should simply be part of `process.env`.

Reply:
It's not a member of `process.env` yet at this point, unless either it could be imported from the environment or `backend:relaunch` was already run, which should not be relied upon in my opinion. 

I've left the possibility to import `GI_VERSION` from the environment just in case, but by default it is supposed to be assigned a dynamic value.

Then I had to find a way to pass it down to the child process, without mutating `process.env` if possible. Hence this code in line 310, although the same was not possible in `backend:launch` so I added `process.env.GI_VERSION = GI_VERSION` there.